tools/assets-automation/asset-sync/README.md

```diff
@@ -593,6 +594,21 @@ So to locally repro this experience:
 6. `pip install -r dev_requirements.txt`
 7. `pytest`
 
+### A note regarding cross-plat usage
+
+The test-proxy utilizes the the `git` of the system running it to retrieve recordings from the assets repository. This means that when loading a recording, the running file system **matters**. When passing a recording path to the test-proxy, ensure that from client side, capitalization is **consistent** across platforms. Let's work through an example.
+
+A test is recorded on `windows`. It writes to relative recording path `a/path/to/recording.json`. On `windows` and `mac`, if a user attempts to start playback for `a/path/To/recording.json`, this would **succeed** at the attempt to load the recording from disk. This is due to the act that the OS is not case-sensitive. On a **linux** system, attempting to load `a/path/To/recording.json` will **fail**.
+
+This is extremely easy to overlook when diagnosing `File not found for playback` issues, as capitalization differences can be difficult to see in context.
```

Comment:
If this becomes a consistent issue, it would be helpful for Test Proxy to spit out possible matches based on case-insensitivity. Still fail but have an exception like

> Unable to find session record `a/path/To/recording.json`, possible match is `a/path/to/recoring.json`. To resolve this issue see <A note regarding cross-plat usage>.